### ISSUE TITLE
Extension tests failing on Linux since 1.5 was released

### ISSUE THREAD

**Issue by DanTup** [Microsoft/vscode#11721]
When the 1.5 release went out our tests on Linux all started failing. Everything still works on Windows/Max, but on Linux the tests seem to just fail.

This build was just before the 1.5 release and worked fine:

https://travis-ci.org/Dart-Code/Dart-Code/builds/158524709

Then I pushed a simple change to the README and updated the version number in package.json and the tests now fail to run:

https://travis-ci.org/Dart-Code/Dart-Code/jobs/158543037

```text
11.12s$ npm test --silent
### VS Code Extension Test Run ###
Current working directory: /home/travis/build/Dart-Code/Dart-Code
Downloading VS Code into "/home/travis/build/Dart-Code/Dart-Code/.vscode-test" from: https://vscode-update.azurewebsites.net/1.5.0/linux-x64/stable
Running extension tests: /home/travis/build/Dart-Code/Dart-Code/.vscode-test/VSCode-linux-x64/code /home/travis/build/Dart-Code/Dart-Code/out/test --extensionDevelopmentPath=/home/travis/build/Dart-Code/Dart-Code --extensionTestsPath=/home/travis/build/Dart-Code/Dart-Code/out/test
Failed to execute tests: Error: spawn EACCES
Tests exited with code: -1
```

I've tried a few rebuilds since, but they all fail the same - I don't think it's even started to run the tests. The same code is building fine on Mac and Windows.

I'll try and do some debugging tomorrow (it's late here and I don't have a Linux box to hand), but thought it was worth raising this in case others could repro or wanted to investigate.

**Comment by Tyriar** [comment]
This is failing due to the switch over from .zip to .tar.gz in https://github.com/Microsoft/vscode-extension-vscode/pull/27. After investigating I've found that while the new extraction function works fine, for some reasons the permissions to actually run the binaries are gone. I believe `gulp-untar` is the culprit.

I have a PR incoming, hopefully this will be published and resolved within the next 6 hours or so. @DanTup apologies for the hassle.

**Comment by bpasero** [comment]
Published 0.11.18, let us know how it goes.

**Comment by DanTup** [comment]
Looks good now; thanks for resolving so quickly! :+1:
